authorizer-app-backend/src/main/java/org/radarbase/authorizer/resources/ProjectResource.kt

```diff
@@ -41,6 +42,7 @@ class ProjectResource(
 
     @GET
     @NeedsPermission(Permission.Entity.PROJECT, Permission.Operation.READ)
+    @Cache(maxAge = 300, isPrivate = true)
```

Comment:
what is the `isPrivate` property?

Reply:
See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control

 > The response may be stored only by a browser's cache, even if the response is normally non-cacheable. If you mean to not store the response in any cache, use no-store instead. This directive is not effective in preventing caches from storing your response.

I’ve understood this directive as: do not cache by proxy servers or on the server side, since the response is authenticated.

Reply:
From the w3c spec https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.9.1:

> private
>> Indicates that all or part of the response message is intended for a single user and MUST NOT be cached by a shared cache. This allows an origin server to state that the specified parts of the response are intended for only one user and are not a valid response for requests by other users. A private (non-shared) cache MAY cache the response.

Reply:
Thanks, that makes sense.